src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcDatabaseMetaDataTest.java

```diff
@@ -820,17 +820,20 @@ public void testGetViews() throws SQLException {
 
   @Test
   public void testGetSchemas() throws SQLException {
-    try (Connection connection = createConnection()) {
+    try (Connection connection = createConnection(env, database)) {
       try (ResultSet rs = connection.getMetaData().getSchemas()) {
         assertThat(rs.next(), is(true));
         assertThat(rs.getString("TABLE_SCHEM"), is(equalTo(DEFAULT_SCHEMA)));
         assertThat(rs.getString("TABLE_CATALOG"), is(equalTo(DEFAULT_CATALOG)));
         assertThat(rs.next(), is(true));
         assertThat(rs.getString("TABLE_SCHEM"), is(equalTo("INFORMATION_SCHEMA")));
         assertThat(rs.getString("TABLE_CATALOG"), is(equalTo(DEFAULT_CATALOG)));
-        assertThat(rs.next(), is(true));
-        assertThat(rs.getString("TABLE_SCHEM"), is(equalTo("SPANNER_SYS")));
-        assertThat(rs.getString("TABLE_CATALOG"), is(equalTo(DEFAULT_CATALOG)));
+        if (!EmulatorSpannerHelper.isUsingEmulator()) {
+          assertThat(rs.next(), is(true));
+          assertThat(rs.getString("TABLE_SCHEM"), is(equalTo("SPANNER_SYS")));
```

Comment:
Just checking is `TABLE_SCHEM` the right intended string?

Reply:
Good point. Yes, this is intentional. That is one of the strange things about the JDBC definition. They decided to abbreviate __one__ of the column names, and did so by removing only the last letter. Or someone made a typo 25 years ago, and it could never be fixed.... I guess we'll never know.